packages/core/core-flows/src/definition/cart/workflows/confirm-variant-inventory.ts

```diff
@@ -110,7 +111,11 @@ export const confirmVariantInventoryWorkflow = createWorkflow(
       return { items }
     })
 
-    confirmInventoryStep(confirmInventoryInput)
+    when({ input }, ({ input }) => {
+      return !input.skipInventoryCheck
+    }).then(() => {
+      confirmInventoryStep(confirmInventoryInput)
+    })
```

Comment:
question: Does it make sense for this check to live inside the `confirmVariantInventoryWorkflow`? Shouldn't it be outside of it when you consume it?

Reply:
Well, this is a good point.
I think it makes sense to separate in a util the portion to format the input to confirm/reserve inventory, from the check.
I will open a follow up PR organizing that.